ex1/ex1/Exercises/Student.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ex1.Exercises
+{
+    public class Student
+    {
+        private string Name { get; set; }
+        private string Surname { get; set; }
+        private string DateOfBirth { get; set; }
```

Comment:
According to the task you should also have defined **Grade** and **Marks** fields/properties.